test/llvm-intrinsics/fshr.ll

```diff
@@ -47,15 +47,15 @@ entry:
 }
 
 ; CHECK-SPIRV: Function [[TYPE_INT_32]] [[NAME_FSHR_FUNC_32]] {{[0-9]+}} [[TYPE_FSHR_FUNC_32]]
-; CHECK-SPIRV: FunctionParameter [[TYPE_INT_32]] [[X_FSHR:[0-9]+]]
-; CHECK-SPIRV: FunctionParameter [[TYPE_INT_32]] [[Y_FSHL:[0-9]+]]
+; CHECK-SPIRV: FunctionParameter [[TYPE_INT_32]] [[X_FSHL:[0-9]+]]
+; CHECK-SPIRV: FunctionParameter [[TYPE_INT_32]] [[Y_FSHR:[0-9]+]]
```

Comment:
When providing the namings, I treated "FSHR" as the name of the whole function. So there are a couple options for this test: renaming both `X_FSHL` back to `_FSHR` to preserve the original meaning or, as a simpler-to-read alternative, renaming both to something like `X_PARAM`/`Y_PARAM`, `X_FUNC`/`Y_FUNC`, etc.